appium/webdriver/extensions/images_comparison.py

```diff
@@ -73,7 +75,8 @@ def match_images_features(self, base64_image1, base64_image2, **opts):
         }
         return self.execute(Command.COMPARE_IMAGES, options)['value']
 
-    def find_image_occurrence(self, base64_full_image, base64_partial_image, **opts):
+    def find_image_occurrence(self, base64_full_image: bytes, base64_partial_image: bytes,
+                              **opts: Any) -> Tuple[bytes, Dict[str, int]]:
```

Comment:
are you sure the returned type is a Tuple? I thought it is a dictionary

Reply:
https://github.com/appium/appium-base-driver/blob/5b12eebcf011cf7676a2987b646a6394ddafb639/lib/basedriver/commands/images.js#L27